src/Nethermind/Nethermind.Optimism/OptimismHeaderValidator.cs

```diff
@@ -40,11 +39,6 @@ public class OptimismHeaderValidator(
 {
     public override bool Validate(BlockHeader header, BlockHeader? parent, bool isUncle, out string? error)
     {
-        if (!OptimismWithdrawals.Validate(specHelper, header, out error))
```

Comment:
Why remove this check? We can validate the header before it goes through the pipeline.

Reply:
It was removed since it's checked in the `OptimismBlockValidator`, particularly since we need to check that `Withdrawals` is the empty list, and `Withdrawals` is a member of `Block`, not `BlockHeader`.